trollmoves/tests/test_ssh_server.py

```diff
@@ -178,6 +179,18 @@ def test_scp_copy(self, mock_scp_client, mock_sshclient):
 
         mocked_scp_client.put.assert_called_once_with(self.origin, urlparse(self.destination_no_port).path)
 
+    @patch('trollmoves.movers.SSHClient', autospec=True)
+    @patch('trollmoves.movers.SCPClient', autospec=True)
+    def test_scp_copy_custom_timeout(self, mock_scp_client, mock_sshclient):
```

Comment:
The timeout isn't tested in this test, so either adjust the name or test the timeout.